accounts-management/errors.go

```diff
@@ -1,4 +1,4 @@
-package core
+package accountsmanagement
```

Comment:
Although this package name makes more sense, it is soooo loong, that I tink `accounts` was good enough.

Wdyt of switching back to `accounts`?

Reply:
On this note, I'd try to get rid of `types` sub-package as well. 
Then I would use things simply like `accounts.Keypair`, `accounts.Account`, etc.

Reply:
Yes, that would be nice, but not doable with this code organization, cause it will be a lot of cyclic dependencies, cause types being used by `accountsmanagement` use `accountsmanagement/types`.